reframe/core/pipeline.py

```diff
@@ -1075,7 +1076,7 @@ def sanity(self):
     def performance(self):
         try:
             self.check_performance()
-        except SanityError:
+        except (PerformanceError, SanityError):
```

Comment:
If `SanityError` is thrown here, it means that a performance value could not be extracted. This is still a sanity error that happens during the performance stage. Since now we have the distinction, it's better to catch only `PerformanceError`.

Reply:
Then `SanityError` will be raised even if the self is not strict